lib/ui/pages/login.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/ephemeral/login_controller.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/shadow_box.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+import 'package:pispapp/ui/widgets/bottom_button.dart';
+import 'package:pispapp/ui/widgets/phone_number.dart';
+
+class Login extends StatelessWidget {
+  Widget _loginWithGoogleTile() {
+    return ListTile(
+      contentPadding: const EdgeInsets.symmetric(),
+      title: const TitleText(
+        text: 'Link Google Account',
+        fontSize: 14,
+      ),
+      trailing: GetBuilder<LoginController>(
+        builder: (_) => _.googleLogin
+            ? Icon(
+                Icons.check_circle_outline,
+              )
+            : GestureDetector(
+                onTap: () async {
+                  await _.onLinkGoogleAccount();
+                },
+                child: Icon(
+                  Icons.keyboard_arrow_right,
+                ),
+              ),
+      ),
+    );
+  }
+
+  Widget _phoneNumberTile() {
```

Comment:
I think it would be more intuitive to prompt user for their phone number in another screen after signing in for the first time using their google account.

Reply:
Will fix in a later pr. Will cause too many merge conflicts on pr's behind this.

Reply:
okay, let's keep this comment unresolved for future reference

Reply:
addressed in https://github.com/mojaloop/pisp-demo-app-flutter/pull/12